tests/scripts/optdata/bootstrap.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Comment:
Does this script basically just copy the project.json file into the appropriate directory in src/.nuget?